api/src/routers/meals.js

```diff
@@ -0,0 +1,77 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const mealsRouter = express.Router();
+
+// Route to return all meals
+mealsRouter.get("/", async (req, res, next) => {
+  try {
+    const meals = await knex("Meal").select("*");
+    res.json(meals);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// Route to insert a new meal
+mealsRouter.post("/", async (req, res, next) => {
+  try {
+    const data = req.body;
+    await knex("Meal").insert(data);
+    res.status(200).json({ message: "created successfully" });
+  } catch (error) {
+    next(error);
+  }
+});
+
+// Route to get a meal by ID
+mealsRouter.get("/:id", async (req, res, next) => {
+  try {
+    const id = req.params.id;
+    const meal = await knex("Meal").select("*").where("id", id);
```

Comment:
`select("*")` is the default, we can live without it :)

Reply:
Thank you, I will remove the select.